src/main/scala/rocket/ALU.scala

```diff
@@ -8,8 +8,7 @@ import chisel3.util.{BitPat, Fill, Cat, Reverse}
 import freechips.rocketchip.config.Parameters
 import freechips.rocketchip.tile.CoreModule
 
-object ALU
-{
+trait ALUFN {
```

Comment:
Can we make this:
```
class ALUFN
...
class ABLUFN extends ALUFN
```

Get rid of the companion object.

Reply:
Done.